docs/transpile/_toc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "title": "Transpile",
+    "children": [
+      {
+        "title": "Introduction",
+        "url": "/transpile"
+      },
+      {
+        "title": "PLACEHOLDER",
+        "url": "/transpile/passmanagers"
+      },
+      {
+        "title": "PLACEHOLDER",
+        "url": "/transpile/defaults_and_configuration_options"
```

Comment:
File should use `-`, not `_`

Reply:
This is based on what @beckykd and @nbronn have for their filename, so it will need to be updated on their PR